CMakeLists.txt

```diff
@@ -161,6 +155,27 @@ option(CHECK_RUNTIME_MALLOC "Check if some memory allocations are performed at r
 
 option(ENABLE_TEMPLATE_INSTANTIATION "Template instantiation of the main library" ON)
 
+set(PINOCCHIO_MODEL_DIR "${PROJECT_SOURCE_DIR}/models")
+
+if(BUILD_BENCHMARK
+   OR BUILD_EXAMPLES
+   OR BUILD_TESTING)
+  # example-robot-data is required
+  if(EXAMPLE_ROBOT_DATA_MODEL_DIR)
+    message(INFO "Using provided example-robot-data at: ${EXAMPLE_ROBOT_DATA_MODEL_DIR}")
```

Comment:
```suggestion
    message(STATUS "Using provided example-robot-data at: ${EXAMPLE_ROBOT_DATA_MODEL_DIR}")
```

Reply:
Also, EXAMPLE_ROBOT_DATA_MODEL_DIR provided by user should be an absolute path.

We can use `file(REAL_PATH <path> <out-var>EXPAND_TILDE)` to ensure that.